openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetAsyncSelectList/DataAssetAsyncSelectList.tsx

```diff
@@ -125,11 +125,16 @@ const DataAssetAsyncSelectList: FC<DataAssetAsyncSelectListProps> = ({
       const { value, reference, displayName } = option;
 
       let label;
-      if (searchIndex === SearchIndex.USER) {
+      if (
+        searchIndex === SearchIndex.USER ||
+        searchIndex.includes('user') ||
+        searchIndex.includes('team')
+      ) {
         label = (
           <Space>
             <ProfilePicture
               className="d-flex"
+              isTeam={reference.type === EntityType.TEAM}
```

Comment:
I feel you can also pass displayName prop here and let ProfilePicture handle everything instead managing here

Reply:
DisplayName is only for avatar and it will not render as a part of the profile picture, so we can't utilize that in this scenario